envergo/moulinette/forms/__init__.py

```diff
@@ -35,7 +35,7 @@ class MoulinetteFormAmenagement(BaseMoulinetteForm):
         min_value=0,
         max_value=10000000,
         help_text="Surface au sol nouvellement impactée par le projet",
-        widget=forms.TextInput(attrs={"placeholder": _("In square meters")}),
+        widget=forms.NumberInput(attrs={"placeholder": _("In square meters")}),
```

Comment:
Attention, je ne sais pas si la situation s'est améliorée mais l'attribut `type=number` [présente de nombreux problèmes est est globalement déconseillé sauf dans certains cas précis](https://technology.blog.gov.uk/2020/02/24/why-the-gov-uk-design-system-team-changed-the-input-type-for-numbers/).

L'input de type text actuel était intentionnel.

[Cela vaudrait peut-être le coup d'appliquer ces recommendations à la place.](https://design-system.service.gov.uk/components/text-input/#numbers)